.ci/run-tests

```diff
@@ -29,14 +29,14 @@ set -euo pipefail
 echo -e "\033[1m>>>>> Start server container >>>>>>>>>>>>>>>>>>>>>>>>>>>>>\033[0m"
 DETACH=true bash $script_path/run-opensearch.sh
 
-if [[ -n "$RUNSCRIPTS" ]]; then
-  for RUNSCRIPT in ${RUNSCRIPTS//,/ } ; do
-    echo -e "\033[1m>>>>> Running run-$RUNSCRIPT.sh >>>>>>>>>>>>>>>>>>>>>>>>>>>>>\033[0m"
-    CONTAINER_NAME=${RUNSCRIPT} \
-      DETACH=true \
-      bash $script_path/run-${RUNSCRIPT}.sh
-  done
-fi
+#if [[ -n "$RUNSCRIPTS" ]]; then
```

Comment:
Wha's this part for? Why comment out this part?

Reply:
Yeah I was testing if It requires anywhere. I'll remove this in the next PR.